scripts/build_shapes.py

```diff
@@ -114,10 +170,13 @@ def get_GADM_layer(country_list, layer_id, geo_crs, update=False, outlogging=Fal
             geo_crs
         )
 
-        # convert country name representation of the main country (GID_0 column)
-        geodf_temp["GID_0"] = [
-            three_2_two_digits_country(twoD_c) for twoD_c in geodf_temp["GID_0"]
-        ]
+        geodf_temp = filter_gadm(
+            geodf=geodf_temp,
+            layer=layer_id,
+            cc=country_code,
+            contended_flag=contended_flag,
+            output_nonstd_to_csv=True,
```

Comment:
Isn't it meant to be `output_nonstd_to_csv=False` in the final version? :)

Reply:
Definitely, thanks for spotting! I forgot to disable it after the debug